ceph/ceph_admin/cephadm_ansible.py

```diff
@@ -27,7 +27,13 @@ def __init__(self, cluster):
             cluster: Ceph cluster object
         """
         self.cluster = cluster
-        self.admin = self.cluster.get_ceph_object("installer")
+        for host in self.cluster.get_nodes():
+            roles = host.role.role_list
+        if "network-node" in roles:
```

Comment:
The roles that are mentioned here have a one to one mapping with the daemon types listed in `ceph`. As `network-node` is not one of them, I would like us to not add this here. This breaks our design.